packages/loader/container-loader/src/container.ts

```diff
@@ -1774,7 +1774,6 @@ export class Container
 	private async createDetached(codeDetails: IFluidCodeDetails) {
 		const attributes: IDocumentAttributes = {
 			sequenceNumber: detachedContainerRefSeqNumber,
```

Comment:
@ChumpChief Can you check these term-related changes?

Reply:
There's also a usage on L1145 that should be deletable?  Basically anything that says OnlyValidTermValue should go away.